config/locales/en/general.yml

```diff
@@ -207,6 +207,7 @@ en:
       contact_us: For technical assistance visit
       copyright: CONSUL, %{year}
       description: This portal uses the %{consul} which is %{open_source}.
+      logo_canada: Symbol of the Government of Canada
```

Comment:
To override the language files we use the same technique as the views.  Please move this code to config/locales/custom/en/general.yml.
You'll need to ensure the entry has the right tree as well.  So in this case the structure in general.yml would look like en: /n footer: /n logo_canada: ....